src/VisualStudio/Core/Def/Implementation/Preview/PreviewUpdater.PreviewDialogWorkspace.cs

```diff
@@ -4,14 +4,14 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Editor.Shared.Preview;
-using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.Preview
 {
     internal partial class PreviewUpdater
     {
-        private class PreviewDialogWorkspace : PreviewWorkspace
+        // internal for testing
```

Comment:
I think we do need the comment here because if it's made private this code will still compile but our private test sources will fail to build.  I don't want to get into a scenario where it looks like this can be made private but we figure out the hard way that it was not the case.